client/my-sites/preview/main.jsx

```diff
@@ -48,11 +48,15 @@ class PreviewMain extends React.Component {
 	debouncedUpdateLayout = debounce( this.updateLayout, 50 );
 
 	componentDidMount() {
-		global.window && global.window.addEventListener( 'resize', this.debouncedUpdateLayout );
+		if ( typeof window !== 'undefined' ) {
+			window && window.addEventListener( 'resize', this.debouncedUpdateLayout );
```

Comment:
Checking the `window` truthiness for a second time is not necessary.

Reply:
I'll remove the second check